tmux.conf

```diff
@@ -104,4 +104,5 @@ bind B set -g prefix ^b
 bind A set -g prefix ^a
 
 # set tmuxline snapshot in vim
-source-file ~/.tmuxline.conf
+if '[ -f ~/.tmuxline.conf.local ]' 'source-file ~/.tmuxline.conf.local'
+if '[ ! -f ~/.tmuxline.conf.local ] && [ -f ~/.tmuxline.conf ]' 'source-file ~/.tmuxline.conf'
```

Comment:
Yeah, I would only check existence for local files prior to sourcing since we include the rest in the setup. If they don't have `~/.tmuxline.conf` after `rake install` something has gone wrong.